falcon/response.py

```diff
@@ -849,6 +845,9 @@ def append_link(
                 Can take values 'anonymous' or 'use-credentials' or None.
                 (See:
                 https://www.w3.org/TR/html50/infrastructure.html#cors-settings-attribute)
+            link_extension(iterable): Provides additional custom attributes, as
+                described in RFC 8288, Section 3.4.2. Each member of the iterable
+                must be a two-membered tuple in the form of (*param*, *value*).
```

Comment:
```suggestion
                must be a two-tuple in the form of (*param*, *value*).
```

also maybe a link to the rfc would not hurt?

Reply:
done!